src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -21,6 +21,9 @@ public enum ErrorCode {
 
   FILTER_UNRESOLVABLE("Unable to resolve the filter"),
 
+  FILTER_MULTIPLE_ID_FILTER(
+      "Should be just have one _id filter, key cannot be restricted by more than one relation if it includes an Equal"),
```

Comment:
Need to reword "be just have one"; maybe "Should only have one _id filter, document id cannot" (I think we refer to 'document id' everywhere, not 'key').